scripts/create-associations.js

```diff
@@ -61,7 +61,7 @@ function prepareQuery() {
     WILDCARD
   ].join('_').toUpperCase();
 
-  return allLineItems;
+  return {"name": allLineItems};
```

Comment:
This makes sense, it's because of a breaking change in `node-google-dfp-wrapper`. Good catch!